src/coreclr/gc/gcpriv.h

```diff
@@ -185,12 +185,9 @@ inline void FATAL_GC_ERROR()
 #define FEATURE_PREMORTEM_FINALIZATION
 #define GC_HISTORY
 
+#ifndef TARGET_WASM
 #define BACKGROUND_GC   //concurrent background GC (requires WRITE_WATCH)
-
-// It is no longer possible to build without BACKGROUND_GC defined, so we have to use this duplicate
-#if !defined(TARGET_WASM) || defined(FEATURE_WASM_THREADS)
```

Comment:
Realixe we don't do threads but is the dropping of `FEATURE_WASM_THREADS` intentional here?

Reply:
Great point, it is, in fact, not.